### DISCUSSION TITLE
Implement XCTest for react-native

### DISCUSSION THREAD

**Discussion by JoenyBui** [callstack/react-native-builder-bob#269] [Q&A]
I've spent days trying to figure this out, but still no dice. I've been trying to write objective-c unit test for my iOS library (using this library to auto-generate) but I'm getting issues with importing and linking react-native correctly.  Does anyone have a working example using this library?  Android unit test setup is pretty straightforward, but XCode has been a giant pain in the ass.

**Comment by sarveshrajan** [comment]
@JoenyBui  Did you manage to find a solution to your problem?  Any Leads would be helpful!
